[2017-07-17 16:46:19] ApiDocsLog.INFO: login cookie: [] []
[2017-07-17 16:49:57] ApiDocsLog.INFO: login cookie: [] []
[2017-07-17 16:54:22] ApiDocsLog.INFO: login cookie: [] []
[2017-07-17 16:55:50] ApiDocsLog.INFO: login cookie: [] []
[2017-07-17 17:08:08] ApiDocsLog.ERROR: fatalErrorCallback! {"status":500,"msg":"服务器错误","data":{"message":"Call to undefined method ApiDocs\\Api\\Docs\\Api::getListA()","file":"E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php","line":60}} []
[2017-07-17 17:09:17] ApiDocsLog.ERROR: fatalErrorCallback! {"status":500,"msg":"服务器错误","data":{"message":"Call to undefined method ApiDocs\\Api\\Docs\\Api::getListA()","file":"E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php","line":60}} []
[2017-07-17 17:23:53] ApiDocsLog.ERROR: fatalErrorCallback! {"status":500,"msg":"服务器错误","data":{"message":"Call to undefined method ApiDocs\\Core\\Db\\Mysql::ypSelect()","file":"E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Api.class.php","line":12}} []
[2017-07-17 17:24:05] ApiDocsLog.ERROR: fatalErrorCallback! {"status":500,"msg":"服务器错误","data":{"message":"Call to undefined method ApiDocs\\Core\\Db\\Mysql::ypSelect()","file":"E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Api.class.php","line":12}} []
[2017-07-17 17:24:56] ApiDocsLog.ERROR: fatalErrorCallback! {"status":500,"msg":"服务器错误","data":{"message":"Call to undefined method ApiDocs\\Core\\Db\\Mysql::ypSelect()","file":"E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Api.class.php","line":12}} []
[2017-07-18 11:15:37] ApiDocsLog.ERROR: catchExceptionCallback! {"status":501,"msg":"服务器错误","data":"databases error:Unknown column 'member_info' in 'where clause' exception 'ApiDocs\\Core\\Exception\\ServerException' with message 'databases error:Unknown column 'member_info' in 'where clause'' in E:\\MiaCode\\api_docs\\Srv\\Core\\Db\\Mysql.class.php:22\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Core\\Db\\Mysql.class.php(37): ApiDocs\\Core\\Db\\Mysql->Select('SELECT * FROM s...')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Struct.class.php(22): ApiDocs\\Core\\Db\\Mysql->GetOne('SELECT * FROM s...')\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\Docs\\Struct->getOneByName()\n#3 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#4 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#5 {main}"} []
[2017-07-18 14:02:38] ApiDocsLog.ERROR: catchExceptionCallback! {"status":301,"msg":"没有指定的数据","data":"Invalid arguments:name. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:name.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:33\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Struct.class.php(43): ApiDocs\\Api\\Base\\ApiBase->checkParam('name')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\Docs\\Struct->checkExists()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-18 14:04:08] ApiDocsLog.ERROR: catchExceptionCallback! {"status":301,"msg":"没有指定的数据","data":"Invalid arguments:name. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:name.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:33\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Struct.class.php(43): ApiDocs\\Api\\Base\\ApiBase->checkParam('name')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\Docs\\Struct->checkExists()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-18 14:04:40] ApiDocsLog.ERROR: catchExceptionCallback! {"status":301,"msg":"没有指定的数据","data":"Invalid arguments:name. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:name.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:33\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Struct.class.php(43): ApiDocs\\Api\\Base\\ApiBase->checkParam('name')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\Docs\\Struct->checkExists()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-18 14:06:19] ApiDocsLog.ERROR: catchExceptionCallback! {"status":301,"msg":"没有指定的数据","data":"Invalid arguments:name. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:name.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:33\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Struct.class.php(43): ApiDocs\\Api\\Base\\ApiBase->checkParam('name')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\Docs\\Struct->checkExists()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-18 14:29:44] ApiDocsLog.ERROR: catchExceptionCallback! {"status":501,"msg":"服务器错误","data":"Invalid service exception 'ApiDocs\\Core\\Exception\\ServerException' with message 'Invalid service' in E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php:35\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(23): ApiDocs\\Api\\Api->_checkApi()\n#1 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#2 {main}"} []
[2017-07-18 14:30:04] ApiDocsLog.ERROR: catchExceptionCallback! {"status":501,"msg":"服务器错误","data":"Invalid service exception 'ApiDocs\\Core\\Exception\\ServerException' with message 'Invalid service' in E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php:35\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(23): ApiDocs\\Api\\Api->_checkApi()\n#1 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#2 {main}"} []
[2017-07-18 14:31:18] ApiDocsLog.ERROR: catchExceptionCallback! {"status":501,"msg":"服务器错误","data":"Invalid service exception 'ApiDocs\\Core\\Exception\\ServerException' with message 'Invalid service' in E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php:35\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(23): ApiDocs\\Api\\Api->_checkApi()\n#1 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#2 {main}"} []
[2017-07-18 14:33:00] ApiDocsLog.ERROR: catchExceptionCallback! {"status":101,"msg":"输入信息为空","data":"Invalid arguments:struct_data. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:struct_data.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:33\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Struct.class.php(55): ApiDocs\\Api\\Base\\ApiBase->checkParam('struct_data')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\Docs\\Struct->addOne()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-18 14:34:31] ApiDocsLog.ERROR: catchExceptionCallback! {"status":101,"msg":"输入信息为空","data":"Invalid arguments:struct_data. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:struct_data.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:33\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Struct.class.php(57): ApiDocs\\Api\\Base\\ApiBase->checkParam('struct_data')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\Docs\\Struct->addOne()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-18 14:40:49] ApiDocsLog.ERROR: catchExceptionCallback! {"status":101,"msg":"输入信息为空","data":"Invalid arguments:struct_data. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:struct_data.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:34\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Struct.class.php(57): ApiDocs\\Api\\Base\\ApiBase->checkParam('struct_data')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\Docs\\Struct->addOne()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-18 14:42:51] ApiDocsLog.ERROR: catchExceptionCallback! {"status":101,"msg":"输入信息为空","data":"Invalid arguments:struct_data. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:struct_data.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:34\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Struct.class.php(57): ApiDocs\\Api\\Base\\ApiBase->checkParam('struct_data')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\Docs\\Struct->addOne()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-18 14:43:55] ApiDocsLog.ERROR: catchExceptionCallback! {"status":101,"msg":"输入信息为空","data":"Invalid arguments:struct_data. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:struct_data.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:34\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Struct.class.php(57): ApiDocs\\Api\\Base\\ApiBase->checkParam('struct_data')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\Docs\\Struct->addOne()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-18 14:48:12] ApiDocsLog.INFO: insert sql:INSERT INTO `struct_info` (`name`,`desp`,`params`) VALUES (\'aaa\',\'aaa\',\'[{\\\"id\\\":1500360255956,\\\"name\\\":\\\"aa\\\",\\\"type\\\":\\\"aa\\\",\\\"desp\\\":\\\"aaa\\\",\\\"is_array\\\":{\\\"state\\\":false},\\\"is_required\\\":{\\\"state\\\":false}}]\') [] []
[2017-07-18 14:49:25] ApiDocsLog.INFO: insert sql:INSERT INTO `struct_info` (`name`,`desp`,`params`) VALUES (\'aaa\',\'aaa\',\'[{\\\"id\\\":1500360255956,\\\"name\\\":\\\"aa\\\",\\\"type\\\":\\\"aa\\\",\\\"desp\\\":\\\"aaa\\\",\\\"is_array\\\":{\\\"state\\\":false},\\\"is_required\\\":{\\\"state\\\":false}}]\') [] []
[2017-07-18 14:50:52] ApiDocsLog.INFO: insert sql:INSERT INTO `struct_info` (`name`,`desp`,`params`) VALUES (\'aaa\',\'aaa\',\'[{\\\"id\\\":1500360255956,\\\"name\\\":\\\"aa\\\",\\\"type\\\":\\\"aa\\\",\\\"desp\\\":\\\"aaa\\\",\\\"is_array\\\":{\\\"state\\\":false},\\\"is_required\\\":{\\\"state\\\":false}}]\') [] []
[2017-07-18 14:51:35] ApiDocsLog.INFO: insert sql:INSERT INTO `struct_info` (`name`,`desp`,`params`) VALUES (\'aaaa\',\'aaa\',\'[{\\\"id\\\":1500360674985,\\\"name\\\":\\\"a\\\",\\\"type\\\":\\\"a\\\",\\\"desp\\\":\\\"a\\\",\\\"is_array\\\":{\\\"state\\\":false},\\\"is_required\\\":{\\\"state\\\":false}}]\') [] []
[2017-07-18 14:53:07] ApiDocsLog.INFO: insert sql:INSERT INTO `struct_info` (`name`,`desp`,`params`) VALUES (\'aaaaa\',\'aaa\',\'[{\\\"id\\\":1500360674985,\\\"name\\\":\\\"a\\\",\\\"type\\\":\\\"a\\\",\\\"desp\\\":\\\"a\\\",\\\"is_array\\\":{\\\"state\\\":false},\\\"is_required\\\":{\\\"state\\\":false}}]\') [] []
[2017-07-18 14:53:45] ApiDocsLog.INFO: insert sql:INSERT INTO `struct_info` (`name`,`desp`,`params`) VALUES (\'b\',\'b\',\'[{\\\"id\\\":1500360815923,\\\"name\\\":\\\"b\\\",\\\"type\\\":\\\"b\\\",\\\"desp\\\":\\\"b\\\",\\\"is_array\\\":{\\\"state\\\":false},\\\"is_required\\\":{\\\"state\\\":false}}]\') [] []
[2017-07-18 14:55:04] ApiDocsLog.INFO: insert sql:INSERT INTO `struct_info` (`name`,`desp`,`params`) VALUES (\'bb\',\'b\',\'[{\\\"id\\\":1500360856341,\\\"name\\\":\\\"b\\\",\\\"type\\\":\\\"b\\\",\\\"desp\\\":\\\"b\\\",\\\"is_array\\\":{\\\"state\\\":false},\\\"is_required\\\":{\\\"state\\\":false}}]\') [] []
[2017-07-18 16:36:03] ApiDocsLog.ERROR: catchExceptionCallback! {"status":101,"msg":"输入信息为空","data":"Invalid arguments:id. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:id.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:33\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Api.class.php(42): ApiDocs\\Api\\Base\\ApiBase->checkParam('id')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\Docs\\Api->checkExists()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-18 16:39:59] ApiDocsLog.ERROR: catchExceptionCallback! {"status":101,"msg":"输入信息为空","data":"Invalid arguments:id. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:id.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:33\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Api.class.php(42): ApiDocs\\Api\\Base\\ApiBase->checkParam('id')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\Docs\\Api->checkExists()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-18 16:42:00] ApiDocsLog.ERROR: catchExceptionCallback! {"status":101,"msg":"输入信息为空","data":"Invalid arguments:id. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:id.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:33\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\Docs\\Api.class.php(42): ApiDocs\\Api\\Base\\ApiBase->checkParam('id')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\Docs\\Api->checkExists()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-18 16:42:39] ApiDocsLog.INFO: insert sql:INSERT INTO `api_info` (`name`,`url`,`desp`,`input_params`,`output_params`) VALUES (\'aa\',\'aa\',\'\',\'[{\\\"id\\\":1500367347859,\\\"name\\\":\\\"a\\\",\\\"type\\\":\\\"a\\\",\\\"desp\\\":\\\"a\\\",\\\"is_array\\\":{\\\"state\\\":false},\\\"is_required\\\":{\\\"state\\\":false}}]\',\'[]\') [] []
[2017-07-19 15:31:51] ApiDocsLog.INFO: insert sql:INSERT INTO `api_info` (`name`,`url`,`desp`,`input_params`,`output_params`) VALUES (\'测试\',\'test\',\'\',\'[{\\\"id\\\":1500449496883,\\\"name\\\":\\\"aaa\\\",\\\"type\\\":\\\"free_coupon\\\",\\\"desp\\\":\\\"\\\\u554a\\\\u554a\\\\u554a\\\\u554a\\\",\\\"is_array\\\":{\\\"state\\\":false},\\\"is_required\\\":{\\\"state\\\":false}}]\',\'[]\') [] []
[2017-07-19 16:04:50] ApiDocsLog.INFO: insert sql:INSERT INTO `api_info` (`name`,`url`,`desp`,`input_params`,`output_params`) VALUES (\'aaaa\',\'aaa\',\'\',\'[{\\\"id\\\":1500451469975,\\\"name\\\":\\\"aa\\\",\\\"type\\\":\\\"aa\\\",\\\"desp\\\":\\\"aa\\\",\\\"is_array\\\":{\\\"state\\\":false},\\\"is_required\\\":{\\\"state\\\":false}}]\',\'[]\') [] []
[2017-07-19 16:17:38] ApiDocsLog.INFO: insert sql:INSERT INTO `api_info` (`name`,`url`,`desp`,`input_params`,`output_params`) VALUES (\'ssss\',\'    aaaab  \',\'\',\'[{\\\"id\\\":1500452230549,\\\"name\\\":\\\"sss\\\",\\\"type\\\":\\\"s\\\",\\\"desp\\\":\\\"s\\\",\\\"is_array\\\":{\\\"state\\\":false},\\\"is_required\\\":{\\\"state\\\":false}}]\',\'[]\') [] []
[2017-07-19 16:20:05] ApiDocsLog.INFO: insert sql:INSERT INTO `api_info` (`name`,`url`,`desp`,`input_params`,`output_params`) VALUES (\'testaaa\',\'testa/lalala\',\'\',\'[{\\\"id\\\":1500452387658,\\\"name\\\":\\\"a\\\",\\\"type\\\":\\\"a\\\",\\\"desp\\\":\\\"a\\\",\\\"is_array\\\":{\\\"state\\\":false},\\\"is_required\\\":{\\\"state\\\":false}}]\',\'[]\') [] []
[2017-07-20 11:12:24] ApiDocsLog.ERROR: catchExceptionCallback! {"status":101,"msg":"输入信息为空","data":"Invalid arguments:password. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:password.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:33\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\User\\Api.class.php(16): ApiDocs\\Api\\Base\\ApiBase->checkParam('password')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\User\\Api->login()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-20 11:13:54] ApiDocsLog.ERROR: catchExceptionCallback! {"status":101,"msg":"输入信息为空","data":"Invalid arguments:password. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:password.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:33\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\User\\Api.class.php(16): ApiDocs\\Api\\Base\\ApiBase->checkParam('password')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\User\\Api->login()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-20 11:25:48] ApiDocsLog.INFO: insert sql:INSERT INTO `user_auth` (`user_id`,`auth_id`,`create_time`,`expire_time`) VALUES (\'1\',\'f7a3a77ccda09cdaafac1f4a49bd9807\',\'2017-07-20 11:25:48\',\'2017-07-27 11:25:48\') [] []
[2017-07-20 11:30:20] ApiDocsLog.INFO: insert sql:INSERT INTO `user_auth` (`user_id`,`auth_id`,`create_time`,`expire_time`) VALUES (\'1\',\'a80034e658bb0a6b6bd46b441693898c\',\'2017-07-20 11:30:20\',\'2017-07-27 11:30:20\') [] []
[2017-07-20 11:30:26] ApiDocsLog.ERROR: catchExceptionCallback! {"status":101,"msg":"输入信息为空","data":"Invalid arguments:username. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:username.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:33\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\User\\Api.class.php(15): ApiDocs\\Api\\Base\\ApiBase->checkParam('username')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\User\\Api->login()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-20 11:30:27] ApiDocsLog.ERROR: catchExceptionCallback! {"status":101,"msg":"输入信息为空","data":"Invalid arguments:username. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:username.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:33\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\User\\Api.class.php(15): ApiDocs\\Api\\Base\\ApiBase->checkParam('username')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\User\\Api->login()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-20 11:36:44] ApiDocsLog.INFO: insert sql:INSERT INTO `user_auth` (`user_id`,`auth_id`,`create_time`,`expire_time`) VALUES (\'1\',\'c1032d01fafb22135ddac61c16b197d0\',\'2017-07-20 11:36:44\',\'2017-07-27 11:36:44\') [] []
[2017-07-20 11:57:18] ApiDocsLog.INFO: insert sql:INSERT INTO `user_auth` (`user_id`,`auth_id`,`create_time`,`expire_time`) VALUES (\'1\',\'41e2f364d2a73de3cede196499a5a5a3\',\'2017-07-20 11:57:18\',\'2017-07-27 11:57:18\') [] []
[2017-07-20 11:59:07] ApiDocsLog.INFO: insert sql:INSERT INTO `user_auth` (`user_id`,`auth_id`,`create_time`,`expire_time`) VALUES (\'1\',\'696113b1819c439156821a918aade87b\',\'2017-07-20 11:59:07\',\'2017-07-27 11:59:07\') [] []
[2017-07-20 12:03:05] ApiDocsLog.INFO: insert sql:INSERT INTO `user_auth` (`user_id`,`auth_id`,`create_time`,`expire_time`) VALUES (\'1\',\'1680b02e733773a161bc5b499f9e44c7\',\'2017-07-20 12:03:05\',\'2017-07-27 12:03:05\') [] []
[2017-07-20 13:18:32] ApiDocsLog.INFO: insert sql:INSERT INTO `user_auth` (`user_id`,`auth_id`,`create_time`,`expire_time`) VALUES (\'1\',\'b419f14e5c118f72fd363307c2e942fe\',\'2017-07-20 13:18:32\',\'2017-07-27 13:18:32\') [] []
[2017-07-20 13:49:24] ApiDocsLog.INFO: insert sql:INSERT INTO `user_auth` (`user_id`,`auth_id`,`create_time`,`expire_time`) VALUES (\'1\',\'5587174f2ef2c431630699ba2d6c9f69\',\'2017-07-20 13:49:24\',\'2017-07-27 13:49:24\') [] []
[2017-07-20 15:18:09] ApiDocsLog.INFO: insert sql:INSERT INTO `user_auth` (`user_id`,`auth_id`,`create_time`,`expire_time`) VALUES (\'1\',\'4671d66f54366284c093116e9a7d1c35\',\'2017-07-20 15:18:09\',\'2017-07-27 15:18:09\') [] []
[2017-07-20 15:25:34] ApiDocsLog.INFO: insert sql:INSERT INTO `user_auth` (`user_id`,`auth_id`,`create_time`,`expire_time`) VALUES (\'1\',\'4e7b960ed7d81e0e1bf888dd02ffaf0a\',\'2017-07-20 15:25:34\',\'2017-07-27 15:25:34\') [] []
[2017-07-20 15:55:37] ApiDocsLog.INFO: insert sql:INSERT INTO `user_auth` (`user_id`,`auth_id`,`create_time`,`expire_time`) VALUES (\'1\',\'1f9448e0ffc9b567124f39f605043a50\',\'2017-07-20 15:55:37\',\'2017-07-27 15:55:37\') [] []
[2017-07-20 16:03:38] ApiDocsLog.INFO: insert sql:INSERT INTO `user_auth` (`user_id`,`auth_id`,`create_time`,`expire_time`) VALUES (\'1\',\'4a369804cbe6c7ecc6d4e7a98eb47cae\',\'2017-07-20 16:03:38\',\'2017-07-27 16:03:38\') [] []
[2017-07-20 16:03:57] ApiDocsLog.INFO: insert sql:INSERT INTO `api_info` (`name`,`url`,`desp`,`input_params`,`output_params`) VALUES (\'生生世世\',\'aaaa\',\'\',\'[{\\\"id\\\":1500537827843,\\\"name\\\":\\\"ffff\\\",\\\"type\\\":\\\"free_groupon\\\",\\\"desp\\\":\\\"\\\\u4e8b\\\\u5b9e\\\\u4e0a\\\",\\\"is_array\\\":{\\\"state\\\":false},\\\"is_required\\\":{\\\"state\\\":false}}]\',\'[]\') [] []
[2017-07-20 16:06:24] ApiDocsLog.INFO: insert sql:INSERT INTO `struct_info` (`name`,`desp`,`params`) VALUES (\'sss\',\'生生世世\',\'[{\\\"id\\\":1500537971776,\\\"name\\\":\\\"aaa\\\",\\\"type\\\":\\\"group_modular_info\\\",\\\"desp\\\":\\\"\\\\u554a\\\\u554a\\\\u554a\\\",\\\"is_array\\\":{\\\"state\\\":false},\\\"is_required\\\":{\\\"state\\\":false}}]\') [] []
[2017-07-20 16:32:51] ApiDocsLog.ERROR: catchExceptionCallback! {"status":101,"msg":"输入信息为空","data":"Invalid arguments:username. exception 'ApiDocs\\Core\\Exception\\ApiException' with message 'Invalid arguments:username.' in E:\\MiaCode\\api_docs\\Srv\\Api\\Base\\ApiBase.class.php:80\nStack trace:\n#0 E:\\MiaCode\\api_docs\\Srv\\Api\\User\\Api.class.php(21): ApiDocs\\Api\\Base\\ApiBase->checkParam('username')\n#1 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(60): ApiDocs\\Api\\User\\Api->login()\n#2 E:\\MiaCode\\api_docs\\Srv\\Api\\Api.class.php(24): ApiDocs\\Api\\Api->_loadApiClass()\n#3 E:\\MiaCode\\api_docs\\Srv\\index.php(12): ApiDocs\\Api\\Api->init()\n#4 {main}"} []
[2017-07-20 16:37:28] ApiDocsLog.INFO: insert sql:INSERT INTO `user_auth` (`user_id`,`auth_id`,`create_time`,`expire_time`) VALUES (\'1\',\'da4341fb4ed152247cbe6ed568795472\',\'2017-07-20 16:37:28\',\'2017-07-27 16:37:28\') [] []
